src/Root/App/Multiplexer/styles.css

```diff
@@ -1,7 +1,10 @@
 .multiplexer {
     display: flex;
     flex-direction: column;
-    width: 100vw;
+    width: 80vw;
```

Comment:
But why?

Reply:
Ramailo, you can go ahead and change it if you like. :P